App/xcbuddykit/Sources/Commands/InitCommand.swift

```diff
@@ -18,9 +18,9 @@ enum InitCommandError: FatalError {
     var description: String {
         switch self {
         case let .alreadyExists(path):
-            return "\(path.asString) already exists"
+            return "⚠️  \(path.asString) already exists"
```

Comment:
What do you think about moving those emojis to the printer? Those characters are nice within the interface that we expose to the user through the printer.

Reply:
Agree 👍